summer-health-app/src/components/AccommodationSearch.tsx

```diff
@@ -0,0 +1,91 @@
+    const [checkOut, setCheckOut] = useState(null);
+    const [number, setNumber] = useState('');
+    const [typeOfAccommodation, setTypeOfAccommodation] = useState('');
+
+    const submitHandler = (event: React.FormEvent) => {
+        event.preventDefault();
+        console.log(destination, checkIn, checkOut, number, typeOfAccommodation);
+    }
+
+    return (
+        <form onSubmit={submitHandler}>
+            <Grid className={classes.accommodation_search} container direction="row" justifyContent="flex-start" alignItems="center">
+                <Grid item>
+                    <TextField className={classes.accommodation_select} select label="Where are you going?" value={destination} 
+                        onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                            setDestination(event.target.value);
+                        }} 
+                        InputProps={{
+                            startAdornment: <InputAdornment position="start"><DirectionsCarIcon/></InputAdornment>,
+                        }}
+                    />
+                </Grid>
+                <Grid item>
+                    <LocalizationProvider dateAdapter={AdapterDateFns}>
+                        <DatePicker
+                            label="Check in"
+                            value={checkIn}
+                            onChange={(newValue) => {
+                                setCheckIn(newValue);
+                            }}
+                            renderInput={(params) => <TextField {...params} />}
+                        />
+                    </LocalizationProvider>
+                </Grid>
+                <Grid item>
+                    <LocalizationProvider dateAdapter={AdapterDateFns}>
+                        <DatePicker
+                            label="Check out"
+                            value={checkOut}
+                            onChange={(newValue) => {
+                                setCheckOut(newValue);
+                            }}
+                            renderInput={(params) => <TextField {...params} />}
+                        />
+                    </LocalizationProvider>
+                </Grid> 
+                <Grid item>
+                    <TextField type="number" label="How many people?" variant="outlined"
+                        value={number} 
+                        onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                            setNumber(event.target.value);
+                        }}
+                        InputProps={{
+                            startAdornment: <InputAdornment position="start"><PersonIcon/></InputAdornment>,
+                        }}
+                    />
+                </Grid>
+                <Grid item>
+                    <TextField className={classes.accommodation_select} select label="What type of accommodation?" value={typeOfAccommodation} 

```

Comment:
Dobivam console error kada otvorim drop down - to mi se dogadja u svim drop downovima...